hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -31,6 +31,11 @@ sudo rpi-eeprom-update -d -f ./pieeprom-new.bin
 sudo reboot
 ```
 
+### Subsequent Bootloader Updates
+
+If you update your bootloader via apt, then any configuration changes made using the process described here *WILL* be migrated to the updated bootloader.
```

Comment:
Probably don't need the emphasis and capitalisation on the 'will' as it makes it look like a scary option, when it's actually the safe option?